templates/firefox-ios.xliff

```diff
@@ -3216,6 +3216,17 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/BottomSheet.strings" datatype="plaintext" source-language="en" target-language="en">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.0.1" build-num="15A507"/>
+    </header>
+    <body>
+      <trans-unit id="Addresses.BottomSheet.UseSavedAddressBottomSheet.v124" xml:space="preserve">
+        <source>Use a saved address?</source>
+        <note>Title displayed on the bottom sheet for prompting users to use a saved address from the list.</note>
```

Comment:
This comment sounds more like a developer comment than a localization one ("on the bottom sheet")
Please avoid the use of tech jargon in localization comments, as most localizers won't understand them

Reply:
@Delphine Thanks, any issues using this comment?
```
This label is used in the address list screen header as a question, prompting the user if they want to use a saved address.
```